src/quo/components/colors/color_picker/view.cljs

```diff
@@ -1,39 +1,66 @@
 (ns quo.components.colors.color-picker.view
   (:require
     [quo.components.colors.color.view :as color]
+    [quo.foundations.colors :as colors]
     [react-native.core :as rn]
     [reagent.core :as reagent]))
 
-(def color-list
-  [:blue :yellow :purple :turquoise :magenta :sky :orange :army :flamingo :camel :copper])
-
 (defn- on-change-handler
   [selected color-name on-change]
   (reset! selected color-name)
   (when on-change (on-change color-name)))
 
-(defn view
+(defn get-item-layout
+  [_ index]
+  #js
+   {:length 48
+    :offset (* (+ 48 8) index)
+    :index  index})
```

Comment:
Can we make the `48` constant inside this `color-picker` component? it would be easier to update it later.